lib/order.rb

```diff
@@ -0,0 +1,28 @@
+class Order
+  attr_reader :menu, :dishes, :quantity, :total
+
+  def initialize(menu_klass)
+    @menu = menu_klass
+    @dishes = []
+  end
+
+  def choose(dish, quantity=1)
+    quantity.times do
+      menu.select { |item,price| dishes.push([item, price]) if item == dish }
+    end
+  end
+
+  def calculate_quantities(order)
+    quantities = Hash.new(0)
+    dishes.each { |num| quantities[num] += 1 }
+    quantities.each { |item, num| puts "Your order contains #{num} x #{item[0]}" }
```

Comment:
Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)